docs/data/tree-view/overview/CustomizedTreeView.tsx

```diff
@@ -1,105 +1,190 @@
+import Collapse from '@mui/material/Collapse';
+import { TransitionProps } from '@mui/material/transitions';
+import { animated, useSpring } from '@react-spring/web';
 
-function MinusSquare(props: SvgIconProps) {
+function DotIcon() {
   return (
-    <SvgIcon fontSize="inherit" style={{ width: 14, height: 14 }} {...props}>
-      {/* tslint:disable-next-line: max-line-length */}
-      <path d="M22.047 22.074v0 0-20.147 0h-20.12v0 20.147 0h20.12zM22.047 24h-20.12q-.803 0-1.365-.562t-.562-1.365v-20.147q0-.776.562-1.351t1.365-.575h20.147q.776 0 1.351.575t.575 1.351v20.147q0 .803-.575 1.365t-1.378.562v0zM17.873 11.023h-11.826q-.375 0-.669.281t-.294.682v0q0 .401.294 .682t.669.281h11.826q.375 0 .669-.281t.294-.682v0q0-.401-.294-.682t-.669-.281z" />
-    </SvgIcon>
+    <Box
+      sx={{
+        width: 6,
+        height: 6,
+        borderRadius: '70%',
+        bgcolor: 'warning.main',
+        display: 'inline-block',
+        verticalAlign: 'middle',
+        zIndex: 1,
+        mr: 1,
+      }}
+    />
   );
 }
 
-function PlusSquare(props: SvgIconProps) {
-  return (
-    <SvgIcon fontSize="inherit" style={{ width: 14, height: 14 }} {...props}>
-      {/* tslint:disable-next-line: max-line-length */}
-      <path d="M22.047 22.074v0 0-20.147 0h-20.12v0 20.147 0h20.12zM22.047 24h-20.12q-.803 0-1.365-.562t-.562-1.365v-20.147q0-.776.562-1.351t1.365-.575h20.147q.776 0 1.351.575t.575 1.351v20.147q0 .803-.575 1.365t-1.378.562v0zM17.873 12.977h-4.923v4.896q0 .401-.281.682t-.682.281v0q-.375 0-.669-.281t-.294-.682v-4.896h-4.923q-.401 0-.682-.294t-.281-.669v0q0-.401.281-.682t.682-.281h4.923v-4.896q0-.401.294-.682t.669-.281v0q.401 0 .682.281t.281.682v4.896h4.923q.401 0 .682.281t.281.682v0q0 .375-.281.669t-.682.294z" />
-    </SvgIcon>
-  );
+declare module 'react' {
+  interface CSSProperties {
+    '--tree-view-color'?: string;
+    '--tree-view-bg-color'?: string;
+  }
 }
 
-function CloseSquare(props: SvgIconProps) {
-  return (
-    <SvgIcon
-      className="close"
-      fontSize="inherit"
-      style={{ width: 14, height: 14 }}
-      {...props}
-    >
-      {/* tslint:disable-next-line: max-line-length */}
-      <path d="M17.485 17.512q-.281.281-.682.281t-.696-.268l-4.12-4.147-4.12 4.147q-.294.268-.696.268t-.682-.281-.281-.682.294-.669l4.12-4.147-4.12-4.147q-.294-.268-.294-.669t.281-.682.682-.281.696 .268l4.12 4.147 4.12-4.147q.294-.268.696-.268t.682.281 .281.669-.294.682l-4.12 4.147 4.12 4.147q.294.268 .294.669t-.281.682zM22.047 22.074v0 0-20.147 0h-20.12v0 20.147 0h20.12zM22.047 24h-20.12q-.803 0-1.365-.562t-.562-1.365v-20.147q0-.776.562-1.351t1.365-.575h20.147q.776 0 1.351.575t.575 1.351v20.147q0 .803-.575 1.365t-1.378.562v0z" />
-    </SvgIcon>
-  );
-}
+type StyledTreeItemProps = TreeItemProps & {
+  labelIcon: React.ElementType;
+  labelText: string;
+};
+
+const StyledTreeItemLabel = styled(Typography)({
+  color: 'inherit',
+  fontFamily: 'General Sans',
```

Comment:
What do you think about standardizing the custom font family we use throughout these customization examples?
I really enjoyed both this one using General Sans and the others using monospace. 😄

Reply:
I'm not sure I know what you mean 🤔 
Whould you use General Sans and monospace in **all** TreeView demos, or you would go just for one of these?

Reply:
I was suggesting that we use just one of the two typefaces for the customized demos that aren't replicating a pre-existing design (i.e., all of them minus the Gmail clone)!